src/bp/core/services/auth/auth-service.ts

```diff
@@ -155,10 +183,30 @@ export default class AuthService {
     return this.users.deleteUser(email, strategy)
   }
 
+  async isTokenVersionValid({ email, strategy, tokenVersion }: TokenUser) {
+    if (email === SERVER_USER) {
```

Comment:
could someone hack this ?

Reply:
The token is validated before this method is called, and a user would need the secret to generate a fake token